t/14-isotovideo.t

```diff
@@ -152,6 +153,36 @@ subtest 'upload assets on demand even in failed jobs' => sub {
     ok(!-e $pool_dir . '/base_state.json',        'no fatal error recorded');
 };
 
+# mock backend/driver
+{
+    package FakeBackendDriver;
+    sub new {
+        my ($class, $name) = @_;
+        my $self = bless({class => $class}, $class);
+        require "backend/$name.pm";
+        $self->{backend} = "backend::$name"->new();
+        return $self;
+    }
+    sub extract_assets {
+        my $self = shift;
+        $self->{backend}->do_extract_assets(@_);
+    }
+}
+
+subtest "upload assets even in an incomplete job " => sub {
```

Comment:
please use single ticks in strings unless you want to evaluate perl variables. Also there is an superfluous space after "job".